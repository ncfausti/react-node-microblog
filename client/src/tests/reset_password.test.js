/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import ResetPassword from '../components/reset_password';

test('Page matches snapshot', () => {
  const component = renderer.create(
    <ResetPassword.WrappedComponent />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
