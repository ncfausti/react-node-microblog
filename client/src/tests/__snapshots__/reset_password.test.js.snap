// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page matches snapshot 1`] = `
<div
  className="root"
>
  <div
    className="topbar"
  >
    <a
      href="/"
      id="title"
    >
      MicroBlog
    </a>
    <div
      className="topbar-right"
    >
      <a
        href="/login"
        id="login"
      >
        Login
      </a>
      <a
        href="/register"
        id="signup"
      >
        Signup
      </a>
    </div>
  </div>
  <div
    id="errMsg"
  >
    <span>
      
    </span>
  </div>
  <div
    className="register"
  >
    <form
      onSubmit={[Function]}
    >
      <label>
        Username
      </label>
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
      <label>
        Verify Your Email Address:
      </label>
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
      <label>
        Reset Your Passsword:
      </label>
      <input
        onChange={[Function]}
        type="password"
        value=""
      />
      <label>
        Repeat Your Passsword:
      </label>
      <input
        onChange={[Function]}
        type="password"
        value=""
      />
      <button
        type="submit"
      >
        Reset Your Password
      </button>
    </form>
  </div>
</div>
`;
